active/SSTI.js

```diff
@@ -0,0 +1,194 @@
+        "velocity": ["#set ($run=9*1371742) $run", "12345678"], // outputs 12345678
+        "Velocity": ["#{set} ($run=9*1371742) $run", "12345678"], // In case 'set' is blacklisted in velocity circumvent like this, outputs 12345678
+        "django": ["{% widthratio 9 1 1371742 %}", "12345678"], // outputs 12345678
+        "Django": ["{% debug %}", "django"], // Check output for "django"
+        "Dot": ["{{=9*1371742}}", "12345678"], //Outputs 12345678
+        "Dust": ['{@math key="9" method="multiply" operand="1371742"/}', "12345678"], // Outputs 12345678
+        "Slim/Jade": ["#{9*1371742}", "12345678"], // Outputs 12345678
+    };
+
+    for (var i in equationExecuteAttacks) {
+
+        // Copy requests before reusing them
+        var fuzzMsg = msg.cloneRequest();
+
+        // Set payload and evidence
+        var payload = equationExecuteAttacks[i][0];
+        var evidence = equationExecuteAttacks[i][1];
+
+        // setParam (message, parameterName, newValue)
+        as.setParam(fuzzMsg, param, payload);
+
+        // sendAndReceive(msg, followRedirect, handleAntiCSRFtoken)
+        as.sendAndReceive(fuzzMsg, false, false);
+        var respBody = fuzzMsg.getResponseBody().toString();
+
+        if (respBody.includes(evidence)) {
+
+            logger("Server Side Template Injection Found! Raising Alert...");
+            raiseAlert(as, fuzzMsg, payload, evidence, 3, param, i);
+            logger("SSTI Expression Evaluation Based Engine Detection Completed...");
+            return;
+        }
+    }
+
+    logger("SSTI Expression Evaluation Based Engine Detection Completed...");
+}
+
+function raiseAlert(as, msg, payload, evidence, confidence, param, engine) {
+
+    var badErrors = ["Infinity", "INF"];
+
+    //Alert variables
+    var pluginId = 1204650;
```

Comment:
Ah, just noticed this - could you reserve a new ID in https://github.com/zaproxy/zaproxy/blob/main/docs/scanners.md at the end of the 100000 section

Reply:
Done.